src/vrdev/src/Scatterplot.ts

```diff
@@ -0,0 +1,57 @@
+//run tsc Scatterplot.ts to compile into Scatterplot.js file
+
+// for running on browser
+// import * as _d3 from "d3";
+
+// declare global {
+//   const d3: typeof _d3;
+// }
+
+//for unit testing
+import * as d3 from "d3";
+
+
+export class Scatterplot{
+    data: number[];
```

Comment:
Let's make these `private`.

Reply:
I agree! If I do this, I will have to call `init ()` in my tests for `generatePts ()` (to access those variables). Is that fine?